_includes/homepage/pre_conference.html

```diff
@@ -106,7 +106,7 @@ <h4 class="expo">Voting</h4>
     {% endif %}
 
     <!-- Proposals -->
-    {% if site.data.conf.talk-proposals.show or site.data.conf.poster-proposals or site.data.conf.panel-proposals.show or site.data.conf.workshop-proposals.show or site.data.conf.shirt-proposals.show or site.data.conf.host-proposals.show or site.data.conf.keynote-proposals.show %}
+    {% if site.data.conf.talk-proposals.show or site.data.conf.panel-proposals.show or site.data.conf.workshop-proposals.show or site.data.conf.shirt-proposals.show or site.data.conf.host-proposals.show or site.data.conf.keynote-proposals.show %}
```

Comment:
@dzoladz I don't think this should have been removed - it ensures that the proposals block is showing if any proposal types are active, including posters.

Reply:
@kaitlinnewson Totally agree... but I couldn't get the proposals block to hide when I set `show: false` in `conf.yml`. I quickly skimmed the code and nothing obvious stood out to me that might be causing the issue. Since I was the one that added the additional check for poster-proposals, I just removed it again. ¯\_(ツ)_/¯ Maybe you can take a look? It would be best to retain the logic for poster-proposals.

Reply:
@dzoladz I think the issue is that it was set as `site.data.conf.poster-proposals` and should have been `site.data.conf.poster-proposals.show`.

Reply:
@kaitlinnewson Ha! Well... that _is_ obvious. I'll fix it in just a bit. Thank you!

Reply:
@kaitlinnewson #96 is available for review and merge